src/webapp/src/auth/AuthenticationOptions.tsx

```diff
@@ -1,7 +1,68 @@
+import { Button } from '@small-ads/ui';
 import React from 'react';
+import styled from 'styled-components';
+import { FacebookSquare } from '@styled-icons/boxicons-logos';
+
+const LoginScreen = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: calc(100vh - 64px);
+`;
+
+const LoginMain = styled.div`
+  align-self: center;
+  flex: 1 1;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const LoginSign = styled.div`
+  font-size: 3rem;
```

Comment:
Are we using px or rem? you said before you will use pxls no?

Reply:
Yes we did, I even did that on purpose. Consistency, consistency, consistency.